test/backend/datastores.athena.spec.js

```diff
@@ -0,0 +1,217 @@
+import {assert} from 'chai';
+
+import {connect} from '../../backend/persistent/datastores/athena';
+
+/* eslint-disable no-invalid-this */
+describe('Athena Unit Tests:', function () {
+    it('Invalid connection strings because incorrect credentials', function(done) {
+        const conn = {
+            region: 'us-east-1',
+            accessKey: 'XXXXXXXX',
+            secretKey: 'XXXXXAAAA',
+            database: 'PLOT.LY-TEST',
+            sqlStatement: 'SELECT * FROM TEST_TABLE LIMIT 100',
+            s3Outputlocation: 's3://aws-athena-query-results-11111111-us-east-1/',
```

Comment:
s/s3Outputlocation/outputS3Bucket/

Reply:
see my comment below about using `beforeEach` (so that you don't need to fix this in every spec).